spec/support/suspenders.rb

```diff
@@ -1,5 +1,5 @@
 module SuspendersTestHelpers
-  APP_NAME = 'dummy'
+  APP_NAME = 'dummy_app'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.